_authentication.md

```diff
@@ -1,7 +1,9 @@
 # Authentication
 Uphold is an OAuth 2.0 compliant service.
 
-Partners looking to integrate with our API must [register an application](#registering-an-application) and use the Web Application Flow. A sample implementation will be available soon.
+Partners looking to integrate with our API must [register an application](#registering-an-application). Applications can either use authorization codes to accept Uphold users, or use client credentials to accept the application owner only.
+
+Applications with authorization codes should use the Web Application Flow described below to accept Uphold users.
```

Comment:
The last sentence contains pretty much the same information as the one before it -- I'd suggest merging them.

Also, the "accept Uphold users" language doesn't sound very intuitive to me. Perhaps something like this would work better:

> Partners looking to integrate with our API must \[register an application](#registering-an-application). Applications that implement a user-facing web interface, to provide custom functionality for multiple Uphold users, should use the \[Web Application Flow](#web-application-flow). Applications that implement a backend interface for a corporate partner (and therefore represent an Uphold user themselves) should use use the \[Client Credentials Flow](#client-credentials-flow).

Thoughts?

Reply:
That sounds much better 👍